src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -47,6 +47,9 @@ const Body = BaseForm.extend(Object.assign(
       const className = this.isOV() ?
         'okta-verify-push-challenge' : ' custom-app-push-challenge';
       this.$el.addClass(className);
+      // Move checkbox below the button
+      const checkbox = this.$el.find('.o-form-fieldset');
```

Comment:
wondering why this is happening.. i see the same code in `postRender` ChallengeAuthenticatorDataOktaVerifyView.js  as well

Reply:
In both views, by default this line https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/internals/BaseForm.js#L40 adds the input before adding any button and other components to the UI. So had to adjust the positioning in postRender. not sure if there is a better way to do it.

Reply:
Nice catch. This use case we never had before where we have to add any input after the button.. even `keep me signed is` is also above `next` or `login` button.. 

But we need to address it properly, instead of this hack.

We should update our BaseForm in a way that it can choose where to add a input.. before a button or after a button..

Default : existing.. always all inputs before the button
Custom: add inputs (some inputs) after the button

Reply:
I will look into this